### PR TITLE
Users/praga/rmvstest

### PR DESCRIPTION
**Pull request by prawalagarwal**

### CODE REVIEW

File: Tasks/VsTest/task.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 75
```
Comment (acesiddhu) [inline Tasks/VsTest/task.json]: make sure we port this to 107 also

File: Tasks/VsTest/vstest.ts [hunk 2]
```diff
@@ -268,7 +275,18 @@ function generateResponseFile(discoveredTests: string): Q.Promise<string> {
     selectortool.arg("GetImpactedtests");
     selectortool.arg("/TfsTeamProjectCollection:" + tl.getVariable("System.TeamFoundationCollectionUri"));
     selectortool.arg("/ProjectId:" + tl.getVariable("System.TeamProject"));
-    selectortool.arg("/buildid:" + tl.getVariable("Build.BuildId"));
+
+    if(context == "CD")
+	{	
+        // Release context. Passing Release Id.
+        selectortool.arg("/buildid:" + tl.getVariable("Release.ReleaseId"));
```
Comment (acesiddhu) [inline Tasks/VsTest/vstest.ts]: >buildid [](start = 27, length = 7)

ideally we should be fixing this. are we still passing this as buildid

**Comment by acesiddhu** [comment]
:shipit:

File: Tasks/VsTest/task.json (hunk 1 above)
Comment (prawalagarwal) [inline Tasks/VsTest/task.json]: yes

File: Tasks/VsTest/vstest.ts [hunk 3]
```diff
@@ -277,6 +295,9 @@ function generateResponseFile(discoveredTests: string): Q.Promise<string> {
     selectortool.arg("/BaseLineFile:" + baseLineBuildIdFile);
     selectortool.arg("/platform:" + platform);
     selectortool.arg("/configuration:" + configuration);
+    selectortool.arg("/releaseuri:" + tl.getVariable("release.releaseUri"));
+    selectortool.arg("/releaseenvuri:" + tl.getVariable("release.environmentUri"));
```
Comment (prawalagarwal) [inline Tasks/VsTest/vstest.ts]: Fixed

File: Tasks/VsTest/vstest.ts (hunk 2 above)
Comment (prawalagarwal) [inline Tasks/VsTest/vstest.ts]: Will open a bug for this

**Pull request merged**